.github/workflows/build_base.yml

```diff
@@ -10,25 +10,34 @@ on:
         required: true
         default: ''
         type: string
+      ARCHITECTURE:
+        description: 'The architecture to build the package for'
+        required: true
+        default: 'x64'
+        type: string
 
   workflow_dispatch:
     inputs:
       CHECKOUT_TO: # This is the branch to checkout to. Defaults to 'master'
         description: 'The branch/tag/commit to checkout to'
         required: true
         default: ''
+      ARCHITECTURE:
+        description: 'The architecture to build the package for'
+        required: true
+        default: 'x64'
+        type: string
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ (inputs.ARCHITECTURE == 'x86_64' || inputs.ARCHITECTURE == 'amd64') && 'ubuntu-latest' || 'wz-linux-arm64' }}
```

Comment:
**nitpick**: the default value is `x64` and I see `x86_64` and `amd64`. Should the default value be one of them?

Reply:
**question**: what is `wz-linux-arm64`? I guess this should be the name of the OS that runs the job and this should be one of availables.

Reply:
> nitpick: the default value is x64 and I see x86_64 and amd64. Should the default value be one of them?

You are correct. It is solved in d4c298cfc9c38ec1c43fe69ec1b0c14d42c4bf70

>  what is wz-linux-arm64? I guess this should be the name of the OS that runs the job and this should be one of those available.

This is the name of the runner that will execute the job. `wz-linux-arm64` is the name of the arm runner that we have available as currently there are no public arm runners.